server/controllers/offer/getSavedOffers.js

```diff
@@ -1,3 +1,10 @@
+const { getSavedOffer } = require('../../database/queries/offers/getSavedOffer.js');
+
 module.exports = (req, res) => {
-  res.send(' get my saved offer');
+  const { memberId } = req.params;
```

Comment:
@alaabadra If / when authentication is built, it would be a good improvement to have this endpoint use the authenticated memberId instead of having memberId come from params.  That would ensure that users could only see their own saved offers, and couldn't manually request saved offers for other users.